go/vt/vtgate/planbuilder/merge.go

```diff
@@ -0,0 +1,131 @@
+package planbuilder
+
+import (
+	"github.com/youtube/vitess/go/vt/sqlparser"
+	"github.com/youtube/vitess/go/vt/vtgate/engine"
+)
+
+// merge builds a Merge primitive
+// used for unions
+type merge struct {
+	Left, Right builder
+	symtab      *symtab
+	emerge      *engine.Merge
+	Colsyms     []*colsym
```

Comment:
Don't need Colsyms. These will be needed only when we decide to allow this opcode in a subquery.